test/end-to-end/test_supervisor_load_does_not_default_to_hab_user.ps1

```diff
@@ -0,0 +1,37 @@
+$username = "hab"
+$password = "Pass@word1"
+net user /add $username $password
+net localgroup administrators $username /add
+Add-Type -TypeDefinition (Get-Content ".expeditor\scripts\end_to_end\LsaWrapper.cs" | Out-String)
+$lsa_wrapper = New-Object -type LsaWrapper
+$lsa_wrapper.SetRight($username, "SeServiceLogonRight")
+
+$env:HAB_ORIGIN = "ci"
+hab origin key generate ci
+hab pkg build test/fixtures/windows_plans/dummy
+. .\results\last_build.ps1
+hab pkg install .\results\$pkg_artifact
+
+hab pkg install core/windows-service
+hab pkg exec core/windows-service install
+Start-Service Habitat
+Wait-Supervisor -Timeout 45
+
+Describe "hab svc load" {
+    $loadOut = hab svc load ci/dummy
```

Comment:
Might be worth another test showing that if you load with a service password that the service _does_ run as `hab`, but that would also require another test service.